pkgs/development/python-modules/torch/default.nix

```diff
@@ -1,5 +1,12 @@
 { stdenv, lib, fetchFromGitHub, fetchpatch, buildPythonPackage, python,
-  config, cudaSupport ? config.cudaSupport, cudaPackages, magma,
+  config, cudaSupport ? config.cudaSupport, cudaPackages,
```

Comment:
similar comment here re `cudaSupport` and `rocmSupport`... IIUC the pytorch build inherits the same constraints as magma?